jsonschema-core.xml

```diff
@@ -1505,8 +1538,9 @@ User-Agent: product-name/5.4.1 so-cool-json-schema/1.0.2 curl/7.43.0
                         </t>
                         <t>
                             Interoperability considerations: See Sections
-                            <xref target="language" format="counter"></xref> and
-                            <xref target="integers" format="counter"></xref> above.
+                            <xref target="language" format="counter"></xref>,
+                            <xref target="integers" format="counter"></xref>, and
+                            <xref target="regex" format="counter"></xref>above.
```

Comment:
does this need a space before "above" ?

Reply:
Ugh, yes.  Fixed in PR #603 if you'd be so kind as to review :-)  Thanks for catching it.